cluster/gce/win1803/k8s-node-setup.psm1

```diff
@@ -877,11 +877,17 @@ function Start-WorkerServices {
       # https://github.com/Microsoft/SDN/blob/master/Kubernetes/windows/start-kubelet.ps1#L117
       # (last checked on 2019-01-07):
       "--pod-infra-container-image=${INFRA_CONTAINER}",
-      "--resolv-conf=`"`"",
-      # The kubelet currently fails when this flag is omitted on Windows.
+
+      # Note that attempting to pass the empty string as '' instead results in --resolv-conf="''".
+      # So we just put nothing after the equals sign. We can't just escape double-quotes,
+      # because even escaped properly here, they trip up sc.exe later.
+      "--resolv-conf=",
+
+      # Both --cgroups-per-qos and --enforce-node-allocatable should be disabled on windows;
+      # the latter requires the former to be enabled to work.
       "--cgroups-per-qos=false",
-      # The kubelet currently fails when this flag is omitted on Windows.
-      "--enforce-node-allocatable=`"`"",
+      "--enforce-node-allocatable=",
```

Comment:
Should we keep all these foo= arguments in one place, and append them later? It's probably easier to ensure consistency down the road.

Reply:
Good point. I consolidated them at the end of the arg list.